### PR TITLE
DSD-886: Adding a11y docs for Feedback components

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-886](https://jira.nypl.org/browse/DSD-886)

## This PR does the following:

- Adds table of contents and a11y docs to Feedback components: `Notification`, `ProgressIndicator`, and `SkeletonLoader`.

## How has this been tested?

Storybook

## Accessibility concerns or updates

- More docs.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr951-z5o4p23ztumcaxqlwfrwb1jkiacd9azi.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6256fba77a1d628954c4ea88

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr951-z5o4p23ztumcaxqlwfrwb1jkiacd9azi.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6256fba77a1d628954c4ea88

File: src/components/SkeletonLoader/SkeletonLoader.stories.mdx [hunk 1]
```diff
@@ -101,7 +113,26 @@ to better view the example. The default value is `100%`.
...
+
+According to W3C's success criterion 2.2.2 for operable user interfaces:
+
+> For any moving, blinking or scrolling information that (1) starts automatically,
+> (2) lasts more than five seconds, and (3) is presented in parallel with other
+> content, there is a mechanism for the user to pause, stop, or hide it unless
+> the movement, blinking, or scrolling is part of an activity where it is essential;
+
+The `SkeletonLoader` component can be used as a placeholder for local content
+that is loaded in parallel but it typically does not last more than five (5)
```
Comment (oliviawongnyc) [inline src/components/SkeletonLoader/SkeletonLoader.stories.mdx:127]: If it lasts more than five seconds would it be in violation of this rule?

File: src/components/Notification/Notification.stories.mdx [hunk 2]
```diff
@@ -118,11 +131,22 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+Every `Notification` component must have a unique `aria-label` attribute. This
+is set through the `ariaLabel` prop. A unique `aria-label` value, along with the
+`aside` HTML landmark element, helps screen readers better navigate a page with
+multiple notification content.
```
Comment (oliviawongnyc) [inline src/components/Notification/Notification.stories.mdx]: "multiple notification content" sounds a little strange to me – maybe just "multiple notifications" or "multiple notification components"?

**Review by oliviawongnyc — APPROVED** [review]
Cool!

File: src/components/SkeletonLoader/SkeletonLoader.stories.mdx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/SkeletonLoader/SkeletonLoader.stories.mdx:127]: Yes.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr951-z5o4p23ztumcaxqlwfrwb1jkiacd9azi.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6256fba77a1d628954c4ea88

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr951-z5o4p23ztumcaxqlwfrwb1jkiacd9azi.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6256fba77a1d628954c4ea88

**Pull request merged**